app/views/core_induction_programmes/modules/edit.html.erb

```diff
@@ -4,6 +4,14 @@
     <h1 class="govuk-heading-l">Content change preview</h1>
     <%= form_with model:@course_module, url: year_module_path, method: :put do |f| %>
       <%= f.govuk_text_field :title, label: { text: "Module title" } %>
+      <%= f.govuk_collection_select(
+            :course_year,
+            @course_module.course_year.core_induction_programme.course_years,
+            :id,
+            :title,
+            label: { text: "Which year would you like to add a module to for?", class: "govuk-heading-m" },
+            hint: { text: "Course Years are listed in order" }
+          ) %>
```

Comment:
Don't think we are adding any modules in edit.

Also, would be nice to have an option to set previous module here too.

Reply:
Yeah, i thought about option for selecting previous module after sorting but definitely can add that in. 

My bad, the label text needs amending on this. It shouldn't say add, select the year you want to assign the module to or along those lines.